hugr-py/src/hugr/hugr/__init__.py

```diff
@@ -1,6 +1,7 @@
 """The main HUGR structure."""
 
 from .base import Hugr, NodeData
+from .composable_pass import ComposablePass
```

Comment:
Open to better suggestions for this module name. I opted to not use `pass.py` as `pass` is a keyword in Python.